# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/workers/WorkerPrivate.h
# Commit: 53705a19f919
# Full Hash: 53705a19f919adf1d104910ac49d4f937d8ffbe8
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2025-04-15 21:21:06
# Regressor Bug: 1899503
# File Overlap Count: 1
# Description:
#   Bug 1899503 - P4 Update SharedWorker windowID for remote worker debugger. r=asuth
#   
#   Depends on D230260
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D231778
# ==============================================================================

diff -r f2debe329bca -r 53705a19f919 dom/workers/WorkerPrivate.h
--- a/dom/workers/WorkerPrivate.h	Tue Apr 15 09:47:39 2025 +0000
+++ b/dom/workers/WorkerPrivate.h	Tue Apr 15 09:47:39 2025 +0000
@@ -1082,6 +1082,8 @@
 
   bool IsQueued() const;
 
+  void UpdateWindowIDToDebugger(const uint64_t& aWindowID, const bool& aIsAdd);
+
   already_AddRefed<WorkerRunnable> MaybeWrapAsWorkerRunnable(
       already_AddRefed<nsIRunnable> aRunnable);
 